docs/localSearchDevelopment.md

```diff
@@ -0,0 +1,33 @@
+# Local search development
+
+You can develop and debug search results (homepage, the "Search for services" field) by running Insights Chrome together with chrome-service-backend.
+
+## Prerequisites
+
+1. Have a go language setup. You can follow the [gmv guide](https://github.com/moovweb/gvm#installing).
+2. Have a podman installed. [Getting started guide](https://podman.io/get-started)
+3. Have the [chrome-service-backend](https://github.com/RedHatInsights/chrome-service-backend) checkout locally.
+4. Make sure you terminal supports the [Makefile](https://makefiletutorial.com/) utility.
+
+## Setting up the development environment
+
+chrome-service-backend is the bridge between kafka and the browser client. It exposes the search-index.json endpoint required for Chrome search to function.
+
+### Run chrome-service-backend first
+
+1. Follow the chrome-service-backend steps for local setup (`make infra` should start the kafka, unleash and db container containers, then run `make dev-static` or `make dev-static-node` to kickstart the chrome service itself).
```

Comment:
You don't need the `make-infra` to run the static endpoints. I should have been more clear before.

The dev-static and dev-static-node are there to avoid the requirement for having the containers running and draining your system while the only thing you need is just a static asset server.